.github/workflows/issue-cmds.yml

```diff
@@ -7,6 +7,7 @@ jobs:
   assignme:
     name: /assignme
     runs-on: ubuntu-latest
+    if: github.event.issue.pull_request && startsWith(github.event.comment.body, '/assignme')
```

Comment:
I do not quite get it: 
Does `github.event.issue.pull_request` mean it has to be a `pull_request` to execute the following steps?
These actions are meant to serve for issues not PRs.

And I do not know yet what you actually encountered. What is it that does not work as expected?

Reply:
Sorry, I should describe the question more clear. See https://github.com/datafuselabs/openraft/actions/runs/1653115173 as an example, any comment that doesn't match the `/assignme` and `/help` command will cause CI to fail. This pr try to fix the problem.

> Does github.event.issue.pull_request mean it has to be a pull_request to execute the following steps? These actions are meant to serve for issues not PRs.

You are right, I make a mistake here, I'll fix it soon.

Reply:
> any comment that doesn't match the /assignme and /help command will cause CI to fail. 

I can't confirm this: The CI includes only a unit test job which you can find it here: https://github.com/datafuselabs/openraft/blob/main/.github/workflows/ci.yaml

This action fails on every comment but should not block a PR CI IMO.
Is there a PR that's blocked by this action?

After all, filtering off non-command would make github comfortable :DDD

Reply:
> Is there a PR that's blocked by this action?

No, it's just that "slash-command-action" failed, I should have used "action failed" instead of "CI failed", sorry for my misrepresentation. =-=